.orchestra/support/revng-container/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -euo pipefail
+
+if [[ -n "${REVNG_CLOUD_GIT_FIFO-}" && -n "${REVNG_CLOUD_S3_FIFO-}" ]]; then
+    while [[ ( ! -e "$REVNG_CLOUD_GIT_FIFO" ) || ( ! -e "$REVNG_CLOUD_S3_FIFO" ) ]]; do
+        sleep 0.1s
+    done
+
+    GIT_MARKER=$(mktemp)
+    S3_MARKER=$(mktemp)
+
+    echo "PULL context $REVNG_DATA_DIR/context $GIT_MARKER" > "$REVNG_CLOUD_GIT_FIFO"
+    echo "PULL begin $REVNG_DATA_DIR/begin $S3_MARKER" > "$REVNG_CLOUD_S3_FIFO"
+
+    while [[ -e "$GIT_MARKER" || -e "$S3_MARKER" ]]; do
+        sleep 0.1s
+    done
+
+    export REVNG_NOTIFY_FIFOS="$REVNG_CLOUD_GIT_FIFO:context,$REVNG_CLOUD_S3_FIFO:begin"
+fi
```

Comment:
* Doc
* Are you sure this belongs here? Is this component `revng-container` or `revng-cloud-container`? If this this works for both situations (i.e., this is useful also for non-cloud situation) this is OK.

Reply:
This is to be run within `minikube` as well as in cloud, dunno if we want to make the distinction or not.
In case I can add the component `revng-cloud-container` that uses the image generated by `revng-container` and adds this file

Reply:
It's fine as long as this still works if one wants to use the container without the cloud.

Reply:
Slightly reworked, now the names are more generic with docs